rules/windows/process_creation/proc_creation_win_commandline_path_traversal_evasion.yml

```diff
@@ -24,8 +24,10 @@ detection:
     selection2:
         CommandLine|contains: '.exe\..\'
     filter:
-        CommandLine|contains: '\Google\Drive\googledrivesync.exe\..\'
+        - CommandLine|contains: '\Google\Drive\googledrivesync.exe\..\'
+        - CommandLine|contains: '\Citrix\Virtual Smart Card\Citrix.Authentication.VirtualSmartcard.Launcher.exe\..\'
```

Comment:
Detection is update so `modified: 2022/09/21` too

Reply:
Thanks, I made that change